spread/build/fedora/task.yaml

```
@@ -1,4 +1,4 @@
-summary: Build (on Fedora)
+summary: Build Fedora packages
 systems: [fedora-*]
 variants: [amd64]
```

Or maybe it's just slow?
```suggestion
variants: [amd64]
kill-timeout: 60m
```